tests/test_views.py

```diff
@@ -708,3 +708,39 @@ def test_get_workflow_logs_unauthorized(app, default_user,
                          query_string={"user": random_user_uuid},
                          content_type='application/json')
         assert res.status_code == 403
+
+
+def test_start_input_parameters(app, session, default_user,
+                                yadage_workflow_with_name):
+    """Test start workflow with inupt parameters."""
+    with app.test_client() as client:
+        os.environ["TESTS"] = "True"
+        # create workflow
+        res = client.post(url_for('api.create_workflow'),
```

Comment:
You can use an already existing fixture like `sample_yadage_workflow_in_db` instead of doing all the creation through the view. Example usage [here](https://github.com/reanahub/reana-workflow-controller/pull/147/files#diff-a4f5c4d64d4811b15e1405573c00f3f4R551).